test/shared/commonFunctionsForTests.sh

```diff
@@ -602,3 +603,95 @@ function score_parallelization_test() {
+        echo ""
+        echo "%%%%% Species not found in the restart file will   %%%%%"
+        echo "%%%%% be bootstrapped (i.e. set to default values) %%%%%"
+    else
+        echo ""
+        echo "%%%%% Integration tests will fail unless all %%%%%"
+        echo "%%%%% species are found in the restart file! %%%%%"
+    fi
+}
+
+
+function gcc_enable_or_disable_bootstrap() {
+    #========================================================================
+    # Edits HEMCO_Config.rc files to enable or disable "bootstrapping",
+    # which is setting missing restart file variables to default values.
+    #
+    # 1st argument: Enable ("yes") or disable ("no") bootstrapping
+    # 2nd argument: Directory containing integration test rundirs
+    #========================================================================
+    bootStrap="${1}"
+    rundirsDir="${2}"
+
+    print_bootstrap_info_message "${bootStrap}"
+
+    # Loop over rundirs
+    for runDir in ${rundirsDir}/*; do
+
+        # Do the following if for only valid GEOS-Chem run dirs
+        expr=$(is_valid_rundir "${runDir}")
+        if [[ "x${expr}" == "xTRUE" ]]; then
+
+            # Specify path HEMCO_Config.rc
+            hcoCfg="${runDir}/HEMCO_Config.rc"
+
+            if [[ "x${bootStrap}" == "xyes" ]]; then
+                # Set missing restart file variables to defaults
+                sed_ie "s/EFYO/CYS/"             "${hcoCfg}"
+                sed_ie "s/EFY.*xyz 1/CYS xyz 1/" "${hcoCfg}"
+                sed_ie "s/EY.*xyz 1/CYS xyz 1/"  "${hcoCfg}"
+            else
+                # Don't set missing restart file variables to defaults
+                sed_ie "s/CYS.*xyz 1/EFY xyz 1/" "${hcoCfg}"
+                sed_ie "s/CYS/EFYO/"             "${hcoCfg}"
+                sed_ie "s/EY.*xyz 1/EFYO xyz 1/" "${hcoCfg}"
+            fi
+        fi
+    done
+}
+
+
+function gchp_enable_or_disable_bootstrap() {
+    #========================================================================
+    # Edits HEMCO_Config.rc files to enable or disable "bootstrapping",
+    # which is setting missing restart file variables to default values.
+    #
+    # 1st argument: Enable ("yes") or disable ("no") bootstrapping
+    # 2nd argument: Directory containing integration test rundirs
+    #========================================================================
+    bootStrap="${1}"
+    rundirsDir="${2}"
+
+    print_bootstrap_info_message "${bootStrap}"
+
+    # Loop over rundirs
+    for runDir in ${rundirsDir}/*; do
+
+        # Do the following if for only valid GEOS-Chem run dirs
+        expr=$(is_gchp_rundir "${runDir}")
+        if [[ "x${expr}" == "xTRUE" ]]; then
+
+            # Specify path HEMCO_Config.rc
+            script="${runDir}/setCommonRunSettings.sh"
+
+            if [[ "x${bootStrap}" == "xyes" ]]; then
+                # Set missing restart file variables to defaults
+                sed_ie "s/_in_Restart=./_in_Restart=0/" "${script}"
```

Comment:
Right now there aren't any other settings that contain "_in_Restarts" but that might not be true in the future. Could you change this to write the full setting name as a precaution? Same for the next block in the if-else.

Reply:
Good catch @lizziel.  I'll fix this.

Reply:
Now fixed in 539fcdd